spyder/plugins/editor/widgets/editor.py

```diff
@@ -2028,6 +2028,10 @@ def current_changed(self, index):
             pass
 
         self.update_plugin_title.emit()
+        # Make sure that any replace happens in the editor on top
+        # See issue 9688

```

Comment:
```suggestion
        # See spyder-ide/spyder#9688
```

Reply:
@oscargus thanks for working on this!

We now have `url awareness` for issues so we now use this syntax for issues on comments so we can `Ctrl/Cmd` click on the text and go to the issue :-)